README.md

```diff
@@ -13,16 +13,11 @@ For more details, please visit our [developer portal](https://developer.kyber.ne
 1. If contracts have not been compiled, run `./compilation.sh`. This step can be skipped subsequently.
 2. Run `./tst.sh`
 3. Use `-f` for running a specific test file.
+4. Use `-k` to specify a specific hardfork version.
 
 ### Example Commands
 `./tst.sh -f "./test/kyberReserve.js"`
-
-## Testing with Istanbul Hardfork
-As of the time of writing, `ganache-cli`  does not yet support the Istanbul hardfork. Kindly perform the following steps to run ganache manually.
-1. Clone the [ganache-cli repo](https://github.com/trufflesuite/ganache-cli/)
-2. CD into the repo and `npm i`
-3. `node cli.js -k istanbul`
-4. In another terminal, cd into this repo. To test against the full suite, simply do `npx buidler test --no-compile`. Otherwise, do `npx buidler test --no-compile {FILE_DIRECTORY}`
+`./tst.sh -f "./test/kyberReserve.js" -k istanbul`
 
```

Comment:
I am using ./tst.sh -k Istanbul
I am not sure it really runs Istanbul.
please print some feed back to show it is in did running istanbul

Reply:
I tried with -k istanbul
and I get no failure with kyberNetwrokProxy.js
so not sure if its running.

Reply:
Wrong logic, needed to switch the statements around. Fork choice is printed if you're running on non-default fork (default is Petersburg)

Reply:
Tests should break because Istanbul fix not included.